src/harness/testcases/WINNF_FT_S_MES_testcase.py

```diff
@@ -0,0 +1,150 @@
+#    Copyright 2016 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import json
+import os
+import unittest
+import sas
+from util import winnforum_testcase
+from datetime import datetime
+
+
+class MeasurementTestcase(unittest.TestCase):
+
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_MES_2(self):
+    """The sas under test to request measurement reporting
+    Register cbsds 6 CBSDs with measCapability make sure the SAS request 
+    The measurement report config after for heartbeat.
+    """
+
+    # Load the devices
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    device_e = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_e.json')))
+    device_f = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_f.json')))
+    device_g = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_g.json')))
+    device_i = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_i.json')))
+    devices = [device_a, device_c, device_e, device_f, device_g, device_i]
+
+    for device in devices:
+        device['measCapability'] = ['RECEIVED_POWER_WITH_GRANT']
+        # Inject FCC and user IDs
+        self._sas_admin.InjectFccId({'fccId': device['fccId']})
+        self._sas_admin.InjectUserId({'userId': device['userId']})
+
+    device_f['measCapability'] = ['RECEIVED_POWER_WITHOUT_GRANT']
+    cbsd_ids = []
+
+    # Register devices
+    request = {'registrationRequest': devices}
+    response = self._sas.Registration(request)['registrationResponse']
+    # Check registration response
+    self.assertEqual(len(response), 6)
+    for resp in response:
+        self.assertTrue('cbsdId' in resp)
+        self.assertEqual(resp['response']['responseCode'], 0)
+        self.assertFalse('RECEIVED_POWER_WITH_GRANT' in resp['measReportConfig'])
```

Comment:
Assane, 

The test code here needs to support 2 cases: 
1. 'measReportConfig' is not returned in the resgistrationResponse object (Since its an optional param). 
2. In case, 'measReportConfig' is returned, 'measReportConfig' does not contain 'RECEIVED_POWER_WITH_GRANT'.